### ISSUE TITLE
Storing Scastie snippets in Github Gists

### ISSUE THREAD

**Issue by markosski** [scalacenter/scastie#431]
I'm not sure where are currently Scastie snippets stored but I would find it useful if I could easily save them to and access form Github Gists. Was this ever considered?

**Comment by OlegYch** [comment]
do you mean exporting to gist?
i guess that should be pretty easy to implement as we already have an option to download full project

**Comment by markosski** [comment]
@OlegYch Yes, but would be nice to also read easily from Gists. I would like to be able to access gist list in Scastie, select one of the scripts to edit/run and then update the gist which gets persisted back in Github.

**Comment by OlegYch** [comment]
that's an interesting idea, you might take a look at https://github.com/scalacenter/scastie/blob/73553b240b0f86d66774f302b0b90f5a331fbc02/storage/src/main/scala/com.olegych.scastie.storage/SnippetsContainer.scala and https://github.com/scalacenter/scastie/blob/73553b240b0f86d66774f302b0b90f5a331fbc02/server/src/main/scala/com.olegych.scastie.web/oauth2/GithubUserSession.scala to see what's involved
